portal/src/texts/profile/picture.md

```diff
@@ -5,303 +5,59 @@ order: 8
 ---
 
 # Bildebruk
--   [Hovedprinsipper](#hovedprinsipper)
--   [Persongalleri](#persongalleri)
--   [Mangfold og variasjon](#mangfold-og-variasjon)
--   [Omgivelser](#omgivelser)
--   [Kategoribilder](#kategoribilder)
-    -   [Skadeforsikring](#skadeforsikring)
-    -   [Bilforsikring](#bilforsikring)
-    -   [Reiseforsikring](#reiseforsikring)
-    -   [Dyreforsikring](#dyreforsikring)
--   [Lys og farger](#lys-og-farger)
--   [Komposisjon](#komposisjon)
--   [Bevegelse](#bevegelse)
--   [Før du tar i bruk bilder](#før-du-tar-i-bruk-bilder)
-    -   [Bilder som bakgrunn](#bilder-som-bakgrunn)
-    -   [Bilder i kort](#bilder-i-kort)
--   [Bildearkiv](#bildearkiv)
-    -   [Kreditering av fotografer](#kreditering-av-fotografer)
-    -   [Bidra med bilder](#bidra-med-bilder)
--   [Oppsummering](#oppsummering)
+Vi bruker bilder som virkemidler til å skape en stemning, understreke et poeng eller hjelpe brukeren med å navigere i våre løsninger. Vi skal bruke bilder fremfor illustrasjoner og ikoner så lenge tematikken ikke blir for tung. Bruk bilder fra vår bildebank når du skal bruke bilder i våre løsninger eller i en presentasjon.
```

Comment:
Kan vi si noe annet en at tematikken blir for tung? Eventuelt bare fjerne det og skrive «Vi skal bruke bilder fremfor illustrasjoner og ikoner.»